src/rtm-handlers.js

```diff
@@ -39,33 +39,36 @@ function onOpenConnection(env) {
 // Check if the message includes penny's ID. If it does, send a message to the draw_it channel
 function onReceiveMessage(env) {
   env.rtm.on(RTM_EVENTS.MESSAGE, function handleRtmMessage(message) {
+    function checkMessage(msg) {
+      return message.text.includes(msg);
+    }
     // TODO: Turn this into a nice ol' switch case
     if (message.text.includes(`<@${env.penny.id}>`)) {
       let responseText = "";
-      if (message.text.includes("give") && message.text.includes("prompt")) {
-        env.rtm.sendMessage(prompts.getRandom(env), "C63GFH05V");
-      } else if (
-        message.text.includes("submit") &&
-        message.text.includes("prompt")
-      ) {
-        let submitPromptText = message.text.substr(27, message.text.length);
+      switch (true) {
+        case checkMessage("give prompt"):
```

Comment:
this looks good, and is cleaner. Eventually we'll be putting more cases into the switch prompt so this will keep things organized.